### ISSUE TITLE
Installation instructions out of date

### ISSUE THREAD

**Issue by CameronDevine** [ros-industrial/packet-simplemessage#24]
The path to place the `packet-simplemessage.lua` script on Linux is out of date. According to the Wireshark documentation, https://www.wireshark.org/docs/wsug_html_chunked/ChPluginFolders.html, the script should be placed in the `~/.local/lib/wireshark/plugins` directory. Therefore, the installation instructions for Linux should read,

```
cd $PACKET_SIMPLEMESSAGE
mkdir -p ~/.local/lib/wireshark/plugins
cp packet-simplemessage.lua ~/.local/lib/wireshark/plugins
```

**Comment by gavanderhoorn** [comment]
The older location still works -- or at least it did last time I checked it -- but the newer locations should indeed be used.

If you could submit a PR that updates the installation instructions I'd merge it.

**Comment by CameronDevine** [comment]
Based on the documentation I referenced above, I agree that it should still work, however I had to move the script to the new location when installing Wireshark today. I will submit a pull request with this change tomorrow

**Comment by gavanderhoorn** [comment]
Fixed by #25.
